examples/animation/custom_skinned_mesh.rs

```diff
@@ -0,0 +1,138 @@
+/// Example taken from https://github.com/KhronosGroup/glTF-Tutorials/blob/master/gltfTutorial/gltfTutorial_019_SimpleSkin.md
+fn main() {
+    App::build()
+        .add_plugins(DefaultPlugins)
+        .insert_resource(AmbientLight {
+            brightness: 1.0,
+            ..Default::default()
+        })
+        .add_startup_system(setup.system())
+        .add_system(joint_animation.system())
+        .run();
+}
+
+struct AnimatedJoint;
+
+fn setup(
+    mut commands: Commands,
+    mut meshes: ResMut<Assets<Mesh>>,
+    mut materials: ResMut<Assets<StandardMaterial>>,
+    mut skinned_mesh_inverse_bindposes_assets: ResMut<Assets<SkinnedMeshInverseBindposes>>,
+) {
+    // Create a camera
+    let mut camera = OrthographicCameraBundle::new_2d();
+    camera.orthographic_projection.near = -1.0;
+    camera.orthographic_projection.far = 1.0;
+    camera.orthographic_projection.scale = 0.005;
+    camera.transform = Transform::from_xyz(0.0, 1.0, 0.0);
+    commands.spawn_bundle(camera);
+
+    // Create inverse bindpose matrices for a skeleton consists of 2 joints
+    let inverse_bindposes =
+        skinned_mesh_inverse_bindposes_assets.add(SkinnedMeshInverseBindposes(vec![
+            Mat4::from_translation(Vec3::new(-0.5, -1.0, 0.0)),
+            Mat4::from_translation(Vec3::new(-0.5, -1.0, 0.0)),
+        ]));
+
+    // Create a mesh
+    let mut mesh = Mesh::new(PrimitiveTopology::TriangleList);
+    mesh.set_attribute(
+        Mesh::ATTRIBUTE_POSITION,
+        vec![
+            [0.0, 0.0, 0.0],
+            [1.0, 0.0, 0.0],
+            [0.0, 0.5, 0.0],
+            [1.0, 0.5, 0.0],
+            [0.0, 1.0, 0.0],
+            [1.0, 1.0, 0.0],
+            [0.0, 1.5, 0.0],
+            [1.0, 1.5, 0.0],
+            [0.0, 2.0, 0.0],
+            [1.0, 2.0, 0.0],
+        ],
+    );
+    mesh.set_attribute(Mesh::ATTRIBUTE_NORMAL, vec![[0.0, 0.0, 1.0]; 10]);
+    mesh.set_attribute(Mesh::ATTRIBUTE_UV_0, vec![[0.0, 0.0]; 10]);
+    mesh.set_attribute(
+        "Vertex_JointIndex",
+        vec![
+            [0, 0, 0, 0],
+            [0, 0, 0, 0],
+            [0, 1, 0, 0],
+            [0, 1, 0, 0],
+            [0, 1, 0, 0],
+            [0, 1, 0, 0],
+            [0, 1, 0, 0],
+            [0, 1, 0, 0],
+            [0, 1, 0, 0],
+            [0, 1, 0, 0],
+        ],
+    );
+    mesh.set_attribute(
+        "Vertex_JointWeight",
+        vec![
+            [1.00, 0.00, 0.0, 0.0],
+            [1.00, 0.00, 0.0, 0.0],
+            [0.75, 0.25, 0.0, 0.0],
+            [0.75, 0.25, 0.0, 0.0],
+            [0.50, 0.50, 0.0, 0.0],
+            [0.50, 0.50, 0.0, 0.0],
+            [0.25, 0.75, 0.0, 0.0],
+            [0.25, 0.75, 0.0, 0.0],
+            [0.00, 1.00, 0.0, 0.0],
+            [0.00, 1.00, 0.0, 0.0],
+        ],
+    );
+    mesh.set_indices(Some(Indices::U16(vec![
+        0, 1, 3, 0, 3, 2, 2, 3, 5, 2, 5, 4, 4, 5, 7, 4, 7, 6, 6, 7, 9, 6, 9, 8,
+    ])));
+
+    // Create joint entities
+    let joint_0 = commands
```

Comment:
Why doesn't this one have an `AnimatedJoint` component?

Reply:
I want to animate only the `joint_1`, so it will be similar to the `gltf_skinned_mesh` example as well as the animation defined in `SimpleSkin.gltf`. What should I comment in this case?

Reply:
I would probably add a comment above the `AnimatedJoint` component to explain that you only want to animate joint 1.